torchaudio/backend/sox_backend.py

```diff
@@ -163,6 +166,9 @@ def save_encinfo(filepath: str,
 
 
 @_mod_utils.requires_module('torchaudio._torchaudio')
+@_mod_utils.deprecated(
+    'Please migrate to "sox_io" backend. See https://github.com/pytorch/audio/issues/903 for the detail',
+    '0.9.0')
```

Comment:
The function is removed, and no substitute is provided, even in #903. What is a user supposed to do? I'd simply encourage them to user the new data format, here or in #903?

Reply:
These are internal structures, which were not supposed to be exposed, therefore there is no replacement. Users can simply use the `save`/`load`/`info` functions. (therefore, `Please migrate to "sox_io" backend`).
I added the description in #903.